fpdf/table.py

```diff
@@ -251,13 +252,32 @@ def render(self):
         self._fpdf.l_margin = prev_l_margin
         self._fpdf.x = self._fpdf.l_margin
 
+    # pylint: disable=too-many-return-statements
     def get_cell_border(self, i, j, cell):
         """
         Defines which cell borders should be drawn.
         Returns a string containing some or all of the letters L/R/T/B,
         to be passed to `fpdf.FPDF.multi_cell()`.
         Can be overriden to customize this logic
         """
+
+        if hasattr(cell, "border"):
+            border = CellBordersLayout.coerce(cell.border)
```

Comment:
Not needed if you perform the call to `.coerce()` in `Row.cell()`

Reply:
I do not think that those `hasattr` & `CellBordersLayout.coerce` calls are needed.

Reply:
I tested on your branch, an lines 264 to 268 can be replaced by:
```python
        if cell.border != CellBordersLayout.INHERIT:
            return str(cell.border)
```